app/src/main/java/com/example/randomusergenerator/user/UserScreen.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.randomusergenerator.user
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.example.randomusergenerator.ui.resources.Dimensions.space_x2
+import com.example.randomusergenerator.user.viewmodel.UserViewModel
+import com.example.randomusergenerator.utils.Constants.NUM_OF_USERS
+
+@Composable
+fun UserScreen() {
+    val userViewModel: UserViewModel = viewModel()
+    LaunchedEffect(key1 = true) {
+        userViewModel.getAllUsers(NUM_OF_USERS)
```

Comment:
SInce the `NUM_OF_USERS` is not controlled from the view, it could be moved to the viewmodel.